tests/lib/config/config-file.js

```diff
@@ -642,7 +642,7 @@ describe("ConfigFile", function() {
             });
         });
 
-        it("should load information from `extends` chain with relative path.", function() {
+        it("should load information from `extends` chain in .eslintrc with relative path.", function() {
```

Comment:
This was a duplicate test name, which caused trouble while I was trying to debug.